### PR TITLE
tests, integration-tests: port unity test to spread

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: tests/main/unity/task.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+summary: Check that a unity snap can start and its window is shown
+
+environment:
+    DISPLAY: ":99.0"
+
+systems: [ubuntu-16.04-64-grub]
```
Comment (mvo5) [inline tests/main/unity/task.yaml]: Very nice and clean to limit to specific systems :)

File: tests/main/unity/task.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    echo "When the app is started"
+    cmd="xvfb-run --server-args=\"$DISPLAY -screen 0 1200x960x24 -ac +extension RANDR\" ubuntu-clock-app.clock"
+    sudo -i -u test /bin/sh -c "$cmd 2>&1 > /dev/null" &
+
+    echo "Then the app window is created"
+    expected=".*?\"qmlscene: clockMainView\": \(\"qmlscene\" \"com\.ubuntu\.clock\"\)"
+    while ! xwininfo -tree -root | grep -Pq "$expected"; do sleep 1; done
+
+    # FIXME: if we the backgrounded job is not stopped here the execute block doesn't end
```
Comment (mvo5) [inline tests/main/unity/task.yaml]: typo: "if we the...", extra *we*.

**Comment by mvo5** [comment]
Very nice! one small typo, otherwise looks fine.

**Comment by fgimenez** [comment]
@mvo5 thank you! :) comment fixed

**Pull request merged**